### DISCUSSION TITLE
Auto Artist add

### DISCUSSION THREAD

**Discussion by richardap1** [RandomNinjaAtk/arr-scripts#68] [Q&A] [answered]
In a previous version I tried to use the auto artist feature but due to multiple root folders, it wasn't possible. However, I was looking at the code and wondered if I modify this line with a static path to the path of one of my root folders would it work?

 [    \"rootFolderPath\": \"$path\",](https://github.com/RandomNinjaAtk/arr-scripts/blob/9e2b4f0774e2822a01d0aaaa79c0d00ecd741ecc/lidarr/AutoArtistAdder.bash#L113C14-L113C14)

Thanks

**Comment by RandomNinjaAtk** [comment, marked as answer]
Manually modifying the scripts is not recommended because they are generally overwritten periodically, basically every-time you restart the container. 

However, on that note, what you stated should likely work... If you do manually modify it.
